haskell-awk.cabal

```diff
@@ -14,13 +14,31 @@ category:       Console
 homepage:       https://github.com/gelisam/hawk#readme
 bug-reports:    https://github.com/gelisam/hawk/issues
 author:         Mario Pastorelli <pastorelli.mario@gmail.com>,  Samuel Gélineau <gelisam@gmail.com>
-maintainer:     Mario Pastorelli <pastorelli.mario@gmail.com>,  Samuel Gélineau <gelisam@gmail.com>
+maintainer:     Samuel Gélineau <gelisam@gmail.com>, Jens Petersen <juhpetersen@gmail.com>
```

Comment:
looks like we forgot to regenerate the `.cabal` file after modifying the `package.yaml` file last time. this means that when we released haskell-awk-1.2, we released without this change. I have updated it via a hackage revision.